tests/models/bert/test_tokenization_bert_tf.py

```diff
@@ -32,6 +32,7 @@ def call(self, inputs):
             return out["pooler_output"]
 
 
+@require_tf
```

Comment:
same question - we don't need this anymore, and can keep it is as in current `main`.